lib/Pipfile

```diff
@@ -49,6 +49,8 @@ python-dateutil = "*"
 enum-compat = "*"
 # python3 code backported to python2
 future = "*"
```

Comment:
Do we need this package?

Reply:
ugh we don't, how did this sneak back in here?